notebooks/02_model/vowpal_wabbit_deep_dive.ipynb

```diff
@@ -0,0 +1,737 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<i>Copyright (c) Microsoft Corporation. All rights reserved.</i>\n",
+    "\n",
+    "<i>Licensed under the MIT License.</i>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<center><h1>Using Vowpal Wabbit for Recommendations</h1>\n",
+    "\n",
+    "<img src=\"https://github.com/VowpalWabbit/vowpal_wabbit/blob/master/logo_assets/vowpal-wabbits-github-logo.png?raw=true\" height=\"30%\" width=\"30%\" alt=\"Vowpal Wabbit\">\n",
+    "</center>\n",
+    "\n",
+    "[Vowpal Wabbit](https://github.com/VowpalWabbit/vowpal_wabbit) is a fast online machine learning library that implements several algorithms relevant to the recommendation use case.\n",
```

Comment:
+1 to this

Reply:
hmmm, i'm not sure if a deep dive into simple regression methods is really all that valuable, but i can put in a little more description of what's happening underneath. My objective here was more about demonstrating functionality within the library vs explaining algorithms that are covered elsewhere. In that way it's a bit different from the other deep dive notebooks.

Reply:
I see your point. If it is not about the algorithm itself, it would be useful to see the advantage of VW implementation. Similar to SVD in Surprise and ALS in Spark, the two algorithms are the same in nature (matrix factorization), but each has its own pros and cons, which make it suit for different scenarios.

Reply:
agree with both, in the case of VW, a deep dive could be focused on what is the suite of algos that VW has, pros&cons, etc, instead of explaining what is logistic regression or linear regression